app/src/main/java/com/example/myfridgehome/dto/Recipe.kt

```diff
@@ -2,6 +2,6 @@ package com.example.myfridgehome.dto
 
 data class Recipe(var title : String = "") {
     override fun toString(): String {
-        return title
+        return "$title"
```

Comment:
Since title is already a String type, these two lines are identical... except the lower line requires more typing.